rest/route/github.go

```diff
@@ -705,12 +708,19 @@ func (gh *githubHookApi) tryDequeueCommitQueueItemForPR(pr *github.PullRequest)
 	return nil
 }
 
-func triggersRetry(action, comment string) bool {
+func triggersPatch(action, comment string) (bool, string) {
```

Comment:
Can you comment this to indicate what and why we return a string value? (Should this even return a boolean now? "" == false)

Reply:
Although actually we could return "" from retryComment so that a patch is created for either manual or automated 👀

Reply:
Can you still add a comment to explain the two return values then? And can we add an explicit patch.AllCallers = ""

Reply:
i added it for retrycomment but not for the other "" because we should be returning something like nil when we return false